src/components/Integrations/Shopify/Shopify.js

```diff
@@ -24,23 +24,14 @@ const Shopify = ({ intl, dependencies: { shopifyClient } }) => {
     }
   `;
 
-  const Breadcrum = ({ className }) => (
-    <span className={className}>
-      <a className="main-section" href={_('common.control_panel_advanced_pref_url')}>
-        {_('common.control_panel')}
-      </a>{' '}
-      <span>&lt;</span>{' '}
-      <span className="secondary-section">{_('common.advanced_preferences')}</span>
-    </span>
+  const Breadcrumb = ({ className }) => (
+    <nav className="dp-breadcrumb">
+      <ul>
+        <li><a href={_('common.control_panel_advanced_pref_url')}>{_('common.control_panel')}</a></li>
+        <li className="active"><span>{_('common.advanced_preferences')}</span></li>
```

Comment:
Is the _span_ necessary? what is the _semantic_ meaning?

Reply:
You are right I can eliminate it, but in the library

Reply:
¿Querés decir que primero hay que hacer el cambio en la librería para luego quitarlo acá?

Si es así, me parece bien esperar el cambio en la librería antes de mergear este PR.

Ojo, tal vez ese elemento sirva para algo, no tengo una opinión fuerte en quietarlo (ya que justamente no se que significa)

Reply:
El `span` no tiene funcionalidad inmediata ni propósito, solo era un elemento para decorar. Agregue la decoración al `li`.

Reply:
Joya, siempre es bueno evitar cosas innecesarias.